.github/workflows/publish_docker_images_cron.yml

```diff
@@ -20,14 +20,14 @@ jobs:
           - latest
           - alpine
           - postgres_12
-          - postgres_13
+          - postgres_14
           - nightly
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.3 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b docker-13-as-main --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
after merging tools PR I will change it

Reply:
I think we can still use v0.8.3 here ?

Reply:
> I think we can still use v0.8.3 here ?

I do not think so. Tools repo decides on which tag will be associated with which Dockerfile, and we updated that logic after v0.8.3